gdk/commands/component/transformer/BuildRecipeTransformer.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import shutil
+from pathlib import Path
+from gdk.common.CaseInsensitive import CaseInsensitiveRecipeFile, CaseInsensitiveDict
```

Comment:
We should start using re exports from `__init__.py` files. On the `gdk.commont.___init__.py` file do

```python
from . CaseInsensitive import *
```

then you could 

```python
from gdk.common import CaseInsensitiveRecipeFile, CaseInsensitiveDict
```

Reply:
Yeah, will make this change in the following PRs.